modules/administration-guide/partials/proc_applying-the-bitbucket-application-link-secret.adoc

```diff
@@ -0,0 +1,63 @@
+:_content-type: PROCEDURE
+:description: Applying the Bitbucket application link Secret
+:keywords: applying-bitbucket, apply-the-bitbucket, apply-bitbucket, apply-secret, applying-secret, apply-a-secret, applying-a-secret, bitbucket, bitbucket-application, bitbucket-app, bitbucket-application-link, bitbucket-app-link, bitbucket-secret, secret
+:navtitle: Applying the Bitbucket application link Secret
+// :page-aliases:
+
+[id="applying-the-bitbucket-application-link-secret_{context}"]
+= Applying the Bitbucket application link Secret
+
+Prepare and apply the Bitbucket application link Secret as follows:
+
+.Prerequisites
+* The preceding procedure, Setting up the Bitbucket application link, has been completed.
+* The following Base64-encoded files that were created during the preceding procedure are prepared:
```

Comment:
Once again, something link, `You have prepared the following Base64-encoded files:`